### PR TITLE
ETS-1447 Add close icon to the repository

### PR DESCRIPTION
**Pull request by BirgitPohl**
Changes the times icon for menus

screenshot 2018-11-12 at 16 15 38 --> screenshot 2018-11-12 at 16 11 21

### CODE REVIEW

**Review by susannegernandt — APPROVED** [review]

File: src/bars.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg width="21px" height="17px" viewBox="0 0 21 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="21px" height="17px" viewBox="0 0 21 17" version="1.1" xmlns="http://www.w3.org/2000/svg">
     <title>burger-menu-bar</title>
```
Comment (brunogeronimo) [inline src/bars.svg]: I know that this file is not part of your changes, but can you please also remove the `title` tag from it? 😄

**Review by brunogeronimo — APPROVED** [review]

File: src/closed.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="21px" height="21px" viewBox="0 0 21 21" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (cllagosr) [inline src/closed.svg]: We should change `times.svg` instead of creating a new file

Comment (BirgitPohl) [reply src/closed.svg]: I'd rather delete that one, what do you think?

Comment (cllagosr) [reply src/closed.svg]: I think `times` is a more generic name

Comment (BirgitPohl) [reply src/closed.svg]: Good point. Changing it.

**Review by dknx01 — APPROVED** [review]

**Review by cllagosr — APPROVED** [review]

**Pull request merged**